### DISCUSSION TITLE
Statistics breakdown about current search

### DISCUSSION THREAD

**Discussion by wolfenmark** [seart-group/ghs#25] [Ideas]
Hi,
maybe it would be useful to have a statistics breakdown after returning a successful search (i.e., how many different languages, average contributors, min-max-average commits just to name a few examples).

An overview of the selected bunch of repositories could help in refining search criteria or have a first glimpse of what's inside a potential dataset even before exporting the output.

In general min, max, average, median, unique values (and/or other usual descriptive statistics) for each 'applicable' output field could be useful for recap/early analysis.

**Comment by wolfenmark** [comment]
Maybe also quartiles and most frequent values could be helpful.

See [this](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.describe.html) for an example of a similar approach in text/table format from Python's Pandas library.

**Comment by emadpres** [comment]
Thanks for the feedback.
Definitely, this is a useful feature and we plan to implement it. But our current implementation already suffers some performance issues when it comes to loading all the results (to download), and providing statistics on the whole results is as expensive as preparing the download file. So as soon as the performance issue is fixed, we get back to this feature and implement it.
